docs/source/quickstart.rst

```diff
@@ -105,8 +117,8 @@ which is by default located at ``<ingest package dir>/logs/ingest.log``.
 View single stage output
 ------------------------
 
-Every ingest stage has the option to write its output to a directory that
-follows this path pattern: ``<ingest package dir>/output/<name of stage>``.
+Ingest stages write their output to directories that follow this path pattern:
```

Comment:
We may want to clarify these outputs are in a tabular and/or JSON format?

Reply:
I want to avoid indicating format because then it needs to change if the format changes.